### PR TITLE
DOCS-10064 Feb 12

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-10064 Feb 12

### CODE REVIEW

**Review by ferferrarazzo — DISMISSED** [review]

File: modules/ROOT/pages/administer-community.adoc [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Administering your API Community Manager community requires performing basic maintenance on your ACM Salesforce organization, checking for important notifications about new ACM package releases and downtime, and updating ACM. Administering your community also includes getting support from MuleSoft if necessary.
+Administering your Anypoint API Community Manager site requires performing
+basic maintenance on your API Community Manager Salesforce organization,
+checking for important notifications about new API Community Manager package
+releases and downtime, and updating API Community Manager. Administering your
+site also includes getting support from MuleSoft if necessary.
```
Comment (dhirsch1) [inline modules/ROOT/pages/administer-community.adoc]:
```suggestion
site also includes getting support from MuleSoft, if necessary.
```

File: modules/ROOT/pages/brand-intro.adoc [hunk 2]
```diff
@@ -1,16 +1,16 @@
-= Brand and Customize Your API Community
+= Brand and Customize Your Site
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Anypoint API Community Manager enables you to customize and adjust the exact details of branding, look and feel, and site structure of your API community, using a simple point-and-click approach.
+Anypoint API Community Manager enables you to customize and adjust the exact details of branding, look and feel, and site structure of your Salesforce Experience Cloud site, using a simple point-and-click approach.
```
Comment (dhirsch1) [inline modules/ROOT/pages/brand-intro.adoc]:
```suggestion
Anypoint API Community Manager enables you to customize and adjust the exact details of branding, look and feel, and the site structure of your Salesforce Experience Cloud site, using a simple point-and-click approach.
```

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -6,12 +6,12 @@ endif::[]
...
 
-API Community Manager is powered by *https://www.salesforce.com/products/community-cloud/overview/[Salesforce Community Cloud, role=external, window=blank]* and provides customization, branding, marketing, and engagement capabilities to serve the different needs of your developer audiences. APIs cataloged in https://docs.mulesoft.com/exchange/[Anypoint Exchange] can be surfaced in API Community Manager for developers to discover them and use them to manage client applications, API access credentials, and consumption metrics.
+API Community Manager is powered by *https://www.salesforce.com/products/experience-cloud/overview/[Salesforce Experience Cloud, role=external, window=blank]* and provides customization, branding, marketing, and engagement capabilities to serve the different needs of your developer audiences. APIs cataloged in https://docs.mulesoft.com/exchange/[Anypoint Exchange] can be surfaced in API Community Manager for developers to discover them and use them to manage client applications, API access credentials, and consumption metrics.
```
Comment (dhirsch1) [inline modules/ROOT/pages/index.adoc]: Is "surfaced" the right word? I haven't seen it used this way.

File: modules/ROOT/pages/moderate-forums.adoc [hunk 4]
```diff
@@ -20,8 +20,8 @@ If the community operator chooses *Delete*, the post is removed and not visible
 
 A community operator can also reply to new discussions and questions, review audit trails, and create and manage forum rules. These rules can filter posts, other content, and members based on criteria defined by the operator. Members matching the criteria can be frozen, and posts and other content matching the criteria can be flagged, blocked, automatically edited to remove banned terms, or held for the operator to review them.
 
-For full information about the Salesforce Community Cloud forums and their moderation options, see the Salesforce documentation.
+For full information about the Salesforce Experience Cloud forums and their moderation options, see the Salesforce documentation.
```
Comment (dhirsch1) [inline modules/ROOT/pages/moderate-forums.adoc]: Suggestion: full > more / full > additional

File: modules/ROOT/pages/update-acm.adoc [hunk 5]
```diff
@@ -184,8 +184,8 @@ Configure sharing settings for guests:
 
 Configure sharing settings for members:
 
-. In *Setup*, search for *Communities* using the *Quick Find* box and click *Communities Settings*.
-. Scroll to *Sharing Sets* and click *Edit* next to the set *[Community Name] Sharing Set*.
+. In *Setup*, search for *Digital Experiences* using the *Quick Find* box and click *Settings*.
```
Comment (dhirsch1) [inline modules/ROOT/pages/update-acm.adoc]:
```suggestion
. In *Setup*, search for *Digital Experiences* using the *Quick Find* box, and click *Settings*.
```

**Review by dhirsch1 — DISMISSED** [review]

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**